ee/packages/omni-lib-ee/src/isDepartmentCreationAvailable.ts

```diff
@@ -0,0 +1,13 @@
+import { License } from '@rocket.chat/license';
+import { isDepartmentCreationAvailable } from '@rocket.chat/omni-lib';
+
+export function isDepartmentCreationAvailablePatch(): void {
+	isDepartmentCreationAvailable.patch(async (next) => {
+		// Skip the standard check when Livechat Enterprise is enabled, as it allows unlimited departments
+		if (License.hasModule('livechat-enterprise')) {
```

Comment:
I vote for keeping the license checks out of the patches 👀

Reply:
yeah, need to define a proper way for that.. this one specifically was just moving the file to the package though